src/main/java/it/pagopa/transactions/projections/handlers/v2/AuthorizationRequestProjectionHandler.java

```diff
@@ -24,7 +26,14 @@ public class AuthorizationRequestProjectionHandler
     private TransactionsViewRepository transactionsViewRepository;
 
     @Override
-    public Mono<Transaction> handle(AuthorizationRequestData data) {
+    public Mono<Transaction> handle(AuthorizationRequestData data) { // temporarily not used
+        return handle(data, Instant.now().toString());
+    }
+
+    public Mono<Transaction> handle(
+                                    AuthorizationRequestData data,
+                                    String creationDate
+    ) {
```

Comment:
Why duplicating method signature? Just create a wrapper record that contains AuthorizationRequestedData and authorization requested event creation date